.travis.yml

```diff
@@ -62,3 +67,10 @@ deploy:
     script: yarn run deploy:$COZY_APP_SLUG
     on:
       tags: true
+# dist: trusty
+sudo: required
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3
```

Comment:
We should use something way more robust than just a sleep 3. 

Something like this : https://github.com/elgalu/docker-selenium/blob/master/xvfb/bin/wait-xvfb.sh